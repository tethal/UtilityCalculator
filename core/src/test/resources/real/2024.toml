[general]
start_date = 2024-01-01
end_date = 2024-12-31
tenant = ["Marie Černá", "Jindřišská 16", "111 50 Praha 1"]
owner = ["Jan Novák", "majitel@example.com"]
report_place = "V Praze"
report_date = 2025-05-08
sources = ["Vyúčtování služeb od Společenství vlastníků za rok 2024"]

[[deposits]]
description="leden - prosinec"
count=12
amount=3000

[[heating]]
annual_cost = 12431.84
start_date = 2024-01-01
end_date = 2024-12-31

[[other_fees]]
start_date = 2024-01-01
end_date = 2024-12-31
annual_cost = 10362.59

[[cold_water.reading]]
meter_id = "1"
reading_date = 2024-01-01
state = 60.7

[[cold_water.reading]]
meter_id = "1"
reading_date = 2025-01-01
state = 102.9

[[cold_water.tariff]]
start_date = 2024-01-01
end_date = 2024-12-31
unit_amount = 140.288151659

[[hot_water.reading]]
meter_id = "1"
reading_date = 2024-01-01
state = 30.7

[[hot_water.reading]]
meter_id = "1"
reading_date = 2025-01-01
state = 51.9

[[hot_water.tariff]]
start_date = 2024-01-01
end_date = 2024-12-31
unit_amount = 122.850943396

[[hot_water.heating_basic_cost]]
start_date = 2024-01-01
end_date = 2024-12-31
annual_cost = 2725.92

[[hot_water.heating_consumable_tariff]]
start_date = 2024-01-01
end_date = 2024-12-31
unit_amount = 257.988
