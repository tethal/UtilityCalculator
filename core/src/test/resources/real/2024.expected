Report:
  dateRange: 2024-01-01...2025-01-01
  tenant: Marie Černá, Jindřišská 16, 111 50 Praha 1
  owner: Jan Novák, majitel@example.com
  placeDate: V Praze, 2025-05-08
  sources: Vyúčtování služeb od Společenství vlastníků za rok 2024
  sections:
    Studená voda: 5920.16
    Teplá voda: 10799.71
    Vytápění: 12431.84
    Ostatní poplatky: 10362.59
    Přijaté zálohy: -36000
    TOTAL: 3514.30

Studená voda:
  readings:
    [1] 2024-01-01...2025-01-01: 60.7...102.9 = 42.2
  priceList:
    2024-01-01...2025-01-01: 42.200 * 140.288151659 = 5920.16

Teplá voda:
  readings:
    [1] 2024-01-01...2025-01-01: 30.7...51.9 = 21.2
  priceList:
    2024-01-01...2025-01-01: 21.200 * 122.850943396 = 2604.44
  heatingBasicParts:
    2024-01-01...2025-01-01: 12.00 * 227.16 = 2725.92
  heatingConsumableParts:
    2024-01-01...2025-01-01: 21.200 * 257.988 = 5469.35

Vytápění:
  2024-01: 1.00 * 12431.84 * 0.19 = 2362.05
  2024-02: 1.00 * 12431.84 * 0.16 = 1989.09
  2024-03: 1.00 * 12431.84 * 0.14 = 1740.46
  2024-04: 1.00 * 12431.84 * 0.09 = 1118.87
  2024-05: 1.00 * 12431.84 * 0.02 = 248.64
  2024-06: 1.00 * 12431.84 * 0.0 = 0.00
  2024-07: 1.00 * 12431.84 * 0.0 = 0.00
  2024-08: 1.00 * 12431.84 * 0.0 = 0.00
  2024-09: 1.00 * 12431.84 * 0.01 = 124.32
  2024-10: 1.00 * 12431.84 * 0.08 = 994.55
  2024-11: 1.00 * 12431.84 * 0.14 = 1740.46
  2024-12: 1.00 * 12431.84 * 0.17 = 2113.41

Ostatní poplatky:
  2024-01-01...2025-01-01: 12.00 * 863.55 = 10362.59

Přijaté zálohy:
  leden - prosinec: 12 * 3000 = 36000
